### ISSUE TITLE
Training speed about Optical Flow Model.

### ISSUE THREAD

**Issue by zhangyuan1994511** [mit-han-lab/temporal-shift-module#53]
Hi, Author.
I set the param 'num_segment=16; batch-size=32' and train the Optical Flow Model on Kinetics dataset. The model can  converg, but the training speed is very slow. Do you have the same question? or how to solve? Looking forward to your reply, thank you very much!

**Comment by tonylins** [comment]
Hi, could you please check if the bottleneck is on data loading or model traninig?

**Comment by zhangyuan1994511** [comment]
@tonylins Yes, I check the code and find the bottleneck is data loading. I find the experiment about Optical Flow Model on Kinetics dataset in your paper also set the same param 'num_segment=16'. Do you have the same question? and any suggestions to solve it? Thank you for your reply!

**Comment by tonylins** [comment]
Hi, I would suggest storing data on an SSD to speed up data loading.
